rdsbroker/broker.go

```diff
@@ -166,6 +175,14 @@ func (b *RDSBroker) Update(instanceID string, details brokerapi.UpdateDetails, a
 	}
 
 	modifyDBInstance := b.modifyDBInstance(instanceID, servicePlan, updateParameters, details)
+
+	if servicePlan.RDSProperties.ConfigureAllocatedStorage && updateParameters.AllocatedStorage > 0 {
```

Comment:
Minor issue: you can omit `servicePlan.RDSProperties.ConfigureAllocatedStorage &&`, as that check is performed within the validation method. Then it would be consistent with [how it is done in the Provision() method](https://github.com/alphagov/paas-rds-broker/pull/29/files#diff-3abf7331e0c58a8e01c3ae65cf78cae8R127).